ospfd/ospfd.c

```diff
@@ -1730,6 +1730,10 @@ int ospf_area_nssa_unset(struct ospf *ospf, struct in_addr area_id)
 	area->no_summary = 0;
 	area->suppress_fa = 0;
 	area->NSSATranslatorRole = OSPF_NSSA_ROLE_CANDIDATE;
+
+	if (area->NSSATranslatorState == OSPF_NSSA_TRANSLATE_ENABLED)
+		ospf_asbr_status_update(ospf, --ospf->redistribute);
+
```

Comment:
Unneeded whitespace.

Reply:
ok

Reply:
Do you mean empty line?

Reply:
Please see issue  https://github.com/FRRouting/frr/issues/17135,.It's a simple corner case, but I don't know how to fix it.